src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -356,6 +356,11 @@ export default (accessToken, userID, opts) => {
       { method: "POST" }
     ),
     lotStandingLoader: gravityLoader("me/lot_standings", { size: 100 }),
+    manageArtworksCollectionsLoader: gravityLoader(
+      "artworks/collections/batch",
+      {},
+      { method: "POST" }
+    ),
```

Comment:
**question (non-blocking):** Just confirming: we are deliberately leaving that second arg on L361 empty because we expect Gravity to enforce authorization based on the access token's current user, correct?